Formula/dashing.rb

```diff
@@ -15,10 +15,8 @@ class Dashing < Formula
 
   depends_on "go" => :build
 
-  # Use ruby docs just as dummy documentation to test with
-  # DO NOT USE https or find a better host, certificate has expired
   resource "ruby_docs_tarball" do
-    url "http://ruby-doc.com/downloads/ruby_2_6_5_core_rdocs.tgz"
+    url "https://ruby-doc.com/downloads/ruby_2_6_5_core_rdocs.tgz"
```

Comment:
Does it work now?

Reply:
It does for me:
```
* Server certificate:
*  subject: CN=ruby-doc.com
*  start date: Jan 17 00:00:00 2020 GMT
*  expire date: Jan 19 23:59:59 2022 GMT
*  subjectAltName: host "ruby-doc.com" matched cert's "ruby-doc.com"
*  issuer: C=GB; ST=Greater Manchester; L=Salford; O=Sectigo Limited; CN=Sectigo RSA Domain Validation Secure Server CA
*  SSL certificate verify ok.
> HEAD / HTTP/1.1
> Host: ruby-doc.com
```

Reply:
I think this is still not fixed for the built-in curl:

```
$ brew reinstall -s dashing
==> Downloading https://ruby-doc.com/downloads/ruby_2_6_5_core_rdocs.tgz

curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
HTTPS-proxy has similar options --proxy-cacert and --proxy-insecure.
Error: Failed to download resource "dashing--ruby_docs_tarball"
Download failed: https://ruby-doc.com/downloads/ruby_2_6_5_core_rdocs.tgz
```

Reply:
For brewed curl, yeah it works fine.

For complete testing, you probably should mark the PR as syntax only because it just skipped the installation and test part.

Reply:
I did add the label `CI-syntax-only`. Do I miss something?

Reply:
Turns out this domain suffers from the same issue that was reported to me in another issue, and turned out to be caused by the 2020-05-30 expiration of a USERTrust intermediate certificate. It was widely reported back then and caused a series of failures all over the internets. Looks like `ruby-doc.com` still didn't fix their cert chain, so it randomly fail with certain clients that have a buggy way (e.g. using GnuTLS, or obsolete versions of OpenSSL) of finding a working certificate chain.